### PR TITLE
Myportfolio

### PR DESCRIPTION
**Pull request by Kaneez98Fatima**
Understand how to parse a Figma design to create a UI.
Flexbox to place elements on the page.
Build a personal portfolio site.
Use images and backgrounds to enhance the look of the website.

### CODE REVIEW

**Review by ZawadiSumba66 — CHANGES_REQUESTED** [review]
#### Hi @Kaneez98Fatima 👋 👋 ,  

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that currently lacks a professional; README file. There is no content in your README and this makes your repo look unprofessional. Please refer to this [template](https://github.com/microverseinc/readme-template) given by Microverse. 🙏 

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: index.html [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+      <a><i class="fa fa-bars cursor"></i></a>
+    </div>
+    </nav>
+  </header>
+  <section class="main-section img">
+    <div>
+      <h1>Hey There.</h1>
+        <h1>I'm Kaneez Fatima</h1>
+        <h2>I'am a Software Developer</h2>
+      </div>
```
Comment (ZawadiSumba66) [inline index.html]: - [ ]  A single page can only contain one `h1` tag. This helps with Search Engine Optimization and makes your code look more semantic. So for the `hey there` and `Im Kaneez Fatima` you can put them in one `h1` tag and separate them with `br` break tag in order to skip to the next line.
- [ ] Kindly increase the font size of the `h1` tag. This is to ensure we replicate the design as much as possible. You can check out the screenshots below for reference.
- Yours
image
 template's
image

**Review by ZawadiSumba66 — CHANGES_REQUESTED** [review]
#### Hi @Kaneez98Fatima again 😃  , 

Good job so far!!

### HIGHLIGHTS 🚀 🚀 
- Professional README file ✔️ 
- Well implemented code ✔️ 
- Great UI design ✔️ 

However, there is one more change you need to consider in order to get to the next step but you are almost there!! 💪 💪 

## Required Changes ♻️
_Check the comments under the review._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 1 above)
Comment (Kaneez98Fatima) [reply index.html]: Yeah Sure @ZawadiSumba66 !! Thank you for your review will do that..

Comment (rloterh) [reply index.html]: Well resolved

**Review by rloterh — APPROVED** [review]
## Status: Approved :heavy_check_mark:

Hi Kaneez, 

Your project is complete! Great job for making the necessary changes recommended by the previous reviewer. There is nothing else to say other than... it's time to merge it 🍾 💯 👍.
Congratulations! 🎉 🎉 🥂


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
__As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have 1 more limited review per this project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).__

**Pull request merged**